Scripts/DCS-BIOS/lib/modules/Module.lua

```diff
@@ -977,6 +977,33 @@ function Module:defineEjectionHandleSwitch(identifier, device_id, command, arg_n
 	return control
 end
 
+--- Adds a new string output containing the frequency of the device
+--- @param identifier string the unique identifier for the control
+--- @param device_id integer the dcs device id
+--- @param max_length integer the maximum length of the string
+--- @param category string the category in which the control should appear
+--- @param description string additional information about the control
+--- @return Control control the control which was added to the module
+function Module:defineFrequencyString(identifier, device_id, max_length, category, description)
+	local alloc = self:allocateString(max_length, identifier)
+	self:addExportHook(function(dev0)
+		local value = tostring(GetDevice(device_id):get_frequency())
+		if value == nil then
+			error("function " .. identifier .. " is sending a nil value from its GetDevice(" .. device_id .. "):get_frequency()")
+		end
+
+		alloc:setValue(value)
```

Comment:
Just out of curiosity - what does this return? When we set frequency, we pass like 3000000 to set 30.00. Does this return the same sort of thing (3000000)? Do we want to at least divide the result by 1000 since we multiply everything by 1000 in the set_frequency calls?

Reply:
Yes it is always multiplied by 1000 from what I have seen.